safegraph/delphi_safegraph/run.py

```diff
@@ -15,50 +16,60 @@
 def run_module():
     """Creates the Safegraph indicator."""
     params = read_params()
+
+    # Place to write output files.
     export_dir = params["export_dir"]
+    # Location of input files.
     raw_data_dir = params["raw_data_dir"]
+
+    # Number of cores to use in multiprocessing.
     n_core = int(params["n_core"])
+
+    # AWS credentials
     aws_access_key_id = params["aws_access_key_id"]
     aws_secret_access_key = params["aws_secret_access_key"]
     aws_default_region = params["aws_default_region"]
     aws_endpoint = params["aws_endpoint"]
-    wip_signal = params["wip_signal"]
+    # Whether to sync `raw_data_dir` with an AWS backend.
+    sync = bool(params["sync"])
```

Comment:
If `params["sync"]` is just getting read in as a str, Python will treat all non-empty str as true:

```
In [1]: bool("false")                                                                                                                                                
Out[1]: True

In [2]: bool("False")                                                                                                                                                
Out[2]: True

In [3]: bool("")                                                                                                                                                     
Out[3]: False
```

Would have to document the params clearly if this is the case. Alternatively could make the key optional the use `.get`, but that also requires equal documentation.

Reply:
I noticed that in development, so I have set the value to be true/false in JSON file.  I'm not sure whether removing the explicit cast to bool makes things better or not.  I can add a check here to verify that the value is bool.

Reply:
I misread one of the examples and thought it was being read in as a str instead of bool (used to be `  "test": "True"`). I think it's fine as is. I think the cast to bool is potentially confusing since it's redundant. Could type annotate or comment that line if it's something we want to highlight